tests/sqlx/README.md

```diff
@@ -139,6 +178,53 @@ QueryAssertion::new(&pool, &sql)
     .await;
 ```
 
+### Helper Functions
+
+Use centralized helpers from `src/helpers.rs`:
+
+```rust
+use eql_tests::{get_ore_encrypted, get_ore_encrypted_as_jsonb};
+
+// Get encrypted ORE value for comparison
+let ore_term = get_ore_encrypted(&pool, 42).await?;
+
+// Get ORE value as JSONB for operations
+let jsonb_value = get_ore_encrypted_as_jsonb(&pool, 42).await?;
+```
+
+## Test Organization
+
+### Test Module Categories
+
+**Operator Tests:**
+- `comparison_tests.rs` - Comparison operators (<, >, <=, >=)
+- `equality_tests.rs` - Equality operators (=, !=)
+- `inequality_tests.rs` - Inequality operators
+- `ore_equality_tests.rs` - ORE-specific equality tests
+- `ore_comparison_tests.rs` - ORE CLLW comparison tests
+- `like_operator_tests.rs` - Pattern matching (LIKE, ILIKE)
+- `containment_tests.rs` - Containment operators (@>, <@)
+- `operator_class_tests.rs` - Operator class definitions
+
+**JSONB Tests:**
+- `jsonb_tests.rs` - JSONB functions and structure validation
+- `jsonb_path_operators_tests.rs` - JSONB path operators
+
+**Infrastructure Tests:**
+- `config_tests.rs` - Configuration management
+- `encryptindex_tests.rs` - Encrypted column creation workflows
+- `aggregate_tests.rs` - Aggregate functions (COUNT, MAX, MIN, GROUP BY)
+- `constraint_tests.rs` - Database constraints on encrypted columns
+- `order_by_tests.rs` - ORDER BY with encrypted data
+
+**Index Tests:**
+- `index_compare_tests.rs` - Index comparison functions (Blake3, HMAC, ORE variants)
+- `operator_compare_tests.rs` - Main compare() function tests
+- `specialized_tests.rs` - Specialized cryptographic functions (STE, ORE, Bloom filter)
+
+**Helpers:**
+- `test_helpers_test.rs` - Tests for test helper functions
+
 ## Comparison to SQL Tests
```

Comment:
This entire section can be removed, as it provides no value to users or Stashies.